### ISSUE TITLE
Comparison with Plover?

### ISSUE THREAD

**Issue by weirdalsuperfan** [codepoke-kk/qwertigraphy#3]
I'm learning Gregg, so this seems pretty awesome, especially for practicing gregg specifically - but it's only for typing, right, not for converting pen strokes to longhand? I'm still looking into it, but I guess maybe [grascii search](https://grascii.readthedocs.io/en/latest/README.html#grascii-search) can do the latter?

On that note, I know with Plover (something I've also been looking into) [you can reach 240-300 wpm](https://www.youtube.com/watch?v=U7AFutd9Gos&list=PLYtyZPMUWeI_4i4pc0peBPGMfSV7C5Nxw&index=2&ab_channel=KennethBurchfiel), so I was wondering how you think this compares. Part of Plover/steno seems to involve not only chording but also "outlines," which seems to basically be what qwertigraphy implements.

I was looking over "why qwertigraphy," but what's the max speed you think you can reach with it? I saw you throwing out numbers like 150 WPM, 250 WPM, and even 600 WPM, but are you saying qwertigraphy can actually go to 250 WPM+? Do you think there's any reason for its max speed to be faster than plover's, in principle (considering plover allows chording)?

Just wondering. I might use both, but I'm trying to nail down what to use when.

Thanks!

**Comment by codepoke-kk** [comment]
Whoops. Sorry for the delay. 

If you want to hit 200+WPM, you should go with Plover. Obviously, I never made even 100+ in my 3 years of using Plover 8+ hours a day, 7 days a week, but that is me being me. If you have the intrinsic gifting and a willingness to drill the skill, go for it!

Using Qwertigraph, I regularly touch 150wpm, but I don't know how fast it can go for a real human beside me. It basically adds 30-50% to your top qwerty typing speed and I can burst to 100wpm raw. So, to hit 250wpm, you would need to hit 180wpm raw speed. That's not me. If it's you, it might be easier for you to hit 250 with Qwertigraph than with Plover, because it requires no core physical retraining. It's just a matter of making the dictionary work for you and typing shorter. 

I have a machine capable of "typing" at 360wpm. The Qwertigraph is able to receive and process that input, giving an output of 600wpm consistently. The thing is, no human is going to type with the regularity of a machine. So somewhere between an input speed of 90wpm (me) and 360wpm (machine) there is going to come a point the Qwertigraph is going to guess wrong on some inputs and flake out. I don't know where that place is. 

Bottom line, if I wanted a quick speed boost and the benefit of pen input, I'd go with the Qwertigraph. If I was invested in max speed and willing to back that investment with sweat, I'd go with Plover. 

As for "reading" pen input, I've got some experimental code out there but it's withered.
